netbout-web/src/test/java/com/netbout/rest/TkAppTest.java

```diff
@@ -155,6 +155,34 @@ public void redirectsToLoginAndSetsCookie() throws Exception {
     }
 
     /**
+     * TkApp includes authenticated user alias in header.
+     * @throws Exception If there is some problem inside
+     * @checkstyle MultipleStringLiteralsCheck (20 lines)
+     */
+    @Test
+    public void hasAuthenticatedUserAliasInHeader() throws Exception {
+        final String loc = "http://example.com/whatever";
+        final String head = new RsPrint(
+            new TkApp(new MkBase()).act(
+                new RqWithHeader(
+                    new RqWithAuth("urn:test:1"),
+                    String.format(
+                        "Cookie: RsReturn=%s",
+                        URLEncoder.encode(loc, Charset.defaultCharset().name())
+                    )
+                )
+            )
+        ).printHead();
+        MatcherAssert.assertThat(
+            "Incorrect X-Netbout-Alias header",
+            Pattern.compile(
+                // @checkstyle LineLengthCheck (1 line)
+                "^X-Netbout-Alias: tester",
```

Comment:
@HDouss  If you start the server locally in the gui there will be an alias called ```tester``` i suspect that this alias is used here to I think this should be returned, @dmzaytsev could you confirm this?